.github/workflows/actions/update-reference-screenshots/action.yml

```diff
@@ -0,0 +1,36 @@
+name: 'Update Reference Screenshots'
+description: 'Update Reference Screenshots'
+
+on:
+  workflow_dispatch:
+
+runs:
+  using: 'composite'
+  steps:
+    - uses: actions/setup-node@v1
+      with:
+        node-version: 15.x
+    - uses: actions/download-artifact@v2
+      with:
+        path: ./artifacts
+    - name: Extract Archives
+      # This finds all .zip files in the ./artifacts
+      # directory, including nested directories.
+      # It then unzips every .zip to the root directory
+      run: |
+        find . -type f -name '*.zip' -exec unzip -q -o -d ../ -- '{}'  -x '*.zip' \;
+      shell: bash
+      working-directory: ./artifacts
+    - name: Push Screenshots
+      # Configure user as Ionitron
+      # and push only the changed .png snapshots
+      # to the remote branch.
+      run: |
+        git config user.name ionitron
+        git config user.email github-actions@github.com
```

Comment:
Is it necessary to set the `user.email` twice here? I cannot recall if this is the portion you were stating needs to be committed by ionitron to trigger tests to fire, but also unsure how setting the `user.email` twice would effect that 🤔

Reply:
The first user.email is old and should be removed.